src/test/java/com/codeus/winter/config/BeanDefinitionFactoryTest.java

```diff
@@ -0,0 +1,27 @@
+package com.codeus.winter.config;
+
+import com.codeus.winter.test.BeanA;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
single import

Reply:
I would recommend to change it in Intellij idea setup
![Screenshot 2025-02-16 at 19 48 20](https://github.com/user-attachments/assets/45fc9f5e-4c53-4a27-81c1-c1491ea4b15a)